framework/source/dom/Dispatcher.js

```diff
@@ -30,10 +30,11 @@ enyo.dispatcher = {
 	windowEvents: ["resize", "load", "unload"],
 	connect: function() {
 		var d = enyo.dispatcher;
-		for (var i=0, e; e=d.events[i]; i++) {
+		var i=0, e;
+		for (; e=d.events[i]; i++) {
```

Comment:
@Tofee This syntax "(;" looks a bit awkward to me? Couldn't we simply drop the var like in L39 below?

Reply:
yes, moving the initializer part out of the for statement isn't very common, though correct js-wise. I could have left "i=0" there I guess.